charts/promtail/values.yaml

```diff
@@ -455,6 +455,31 @@ config:
           {{- toYaml . | nindent 4 }}
           {{- end }}
 
+    tracingConfigs: |
+      # Current only support configure via enviroment variables
+      # See also https://www.jaegertracing.io/docs/1.16/client-features/
+      enabled: false
+      JAEGER_SERVICE_NAME: ""
+      JAEGER_TAGS: ""                         # hostname=foobar,my.app.version=1.2.3
+      JAEGER_DISABLED: false
+      JAEGER_AGENT_HOST: ""
+      JAEGER_AGENT_PORT: ""
+      JAEGER_ENDPOINT: ""
+      JAEGER_AUTH_TOKEN: ""
+      JAEGER_USER: ""
+      JAEGER_PASSWORD: ""
+      JAEGER_REPORTER_LOG_SPANS: ""
+      JAEGER_REPORTER_MAX_QUEUE_SIZE: 0
+      JAEGER_REPORTER_FLUSH_INTERVAL: 0
+      JAEGER_SAMPLER_TYPE: ""                 # const, probabilistic
+      JAEGER_SAMPLER_PARAM: ""                # probability=0.001
+      JAEGER_SAMPLER_MANAGER_HOST_PORT: ""
+      JAEGER_SAMPLER_REFRESH_INTERVAL: 0
+      JAEGER_SAMPLER_MAX_OPERATIONS: 0
+      JAEGER_PROPAGATION: "w3c"               # b3, jaeger, w3c
+      JAEGER_TRACEID_128BIT: false
+      JAEGER_RPC_METRICS: false
```

Comment:
I'd recommend removing the above - this should be set as env vars by the user, no need to configure the defaults here since that will be handled by the Golang Jaeger client.